library/include/modules/MapCache.h

```diff
@@ -55,7 +55,7 @@ class Block;
 
 struct BiomeInfo {
     // Determined by the 4-bit index in the designation bitfield
-    static const unsigned MAX_LAYERS = 16;
+    static constexpr unsigned MAX_LAYERS = 16;
```

Comment:
Cool trick
I didn't know that constexpr is equivalent to inline for static variables